packages/docs/package.json

```diff
@@ -12,8 +12,10 @@
   "dependencies": {
     "@docsearch/css": "^3.3.2",
     "@docsearch/react": "3.3.2",
-    "@fuel-ui/css": "^0.10.1",
-    "@fuel-ui/react": "^0.10.1",
+    "@fuel-ui/config": "^0.10.2",
+    "@fuel-ui/css": "^0.10.2",
+    "@fuel-ui/react": "^0.10.2",
+    "@fuel-ui/test-utils": "^0.10.2",
```

Comment:
this dependency change were made because of the linking recursive, right?

the projects don't really need that. can we revert?